docs/application/web/guides/device/system-information.md

```diff
@@ -234,6 +234,31 @@ To receive notifications on property value changes:
    var id = tizen.systeminfo.addPropertyValueArrayChangeListener('SIM', successCallback);
    ```
 
+
+<b>Remark:</b>
+
+In case of `WIFI_NETWORK` property, value change listener is triggered on `ipAddress` and `ip6Address` properties change in the network layer. These changes are not consistent with the `status` or `signalStrength` properties of a physical adapter in physical layer.
+
+According to the previous constraints, in a specific situation, the listener could be triggered just before network adapter shutdown. In this case, the value of `status` returned by the listener will be outdated.
+
+To ensure receiving proper values, use the following code example:
+
+   ```
```

Comment:
Please keep the code indentation. I think the following writing style is better.

To ensure receiving proper values, use the following code example:
 ```
function errorCallback(err) {
        console.log('Error occurred ' + err.code)
};

...
```

Reply:
There is indentation of 4 spaces in whole file. I think, that it's better to keep it consistent.

Reply:
I think you don't understand my comment.  
To indent with line 242, you have to align code from column 1 because the paragraph start from column 1.

However, The code on line 226 has the different indentation from line 242 because it is sub paragraph of 3. item.

According to style guide, you have to write code as follow:

## Section Title

1st paragraph
```
code
```

1. 1st item

    paragraph

2. 2nd item    

    paragraph

3.  3nd item
    
    paragraph 	 
    
    ```    
    code
    ```  

2nd paragraph
```    
code
```

===============

Markdown of above code is as follow:
```
## Section Title

1st paragraph
\`\`\`
code
\`\`\`

1. 1st item

    paragraph

2. 2nd item    

    paragraph

3.  3nd item
    
    paragraph 	 
    
    \`\`\`
    code
    \`\`\`

2nd paragraph
\`\`\`
code
\`\`\`

```

Reply:
You are right, I misunderstood your comment. I think I have fixed the indentation of code example as you suggested.